### PR TITLE
fixup! rename NodeAttr to StableAttr

### PR DESCRIPTION
**Pull request by navytux**
I saw build error when running tests:

	(neo) (z-dev) (g.env) kirr@deco:~/src/neo/src/github.com/hanwen/go-fuse$ go test ./...
	# github.com/hanwen/go-fuse/nodefs
	nodefs/inode.go:130:30: undefined: NodeAttr
	FAIL    github.com/hanwen/go-fuse/benchmark [build failed]
	...

### CODE REVIEW

File: nodefs/inode.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func (n *Inode) setEntryOut(out *fuse.EntryOut) {
 }
 
 // StableAttr returns the (Ino, Gen) tuple for this node.
-func (n *Inode) StableAttr() NodeAttr {
+func (n *Inode) StableAttr() StableAttr {
```
Comment (hanwen) [inline nodefs/inode.go]: aha, perl -i~ -pe s/a/b/ only does one substitution per line :-/

**Comment by hanwen** [comment]
d'oh. I did some more renaming. And tested before pushing. :)

File: nodefs/inode.go (hunk 1 above)
Comment (navytux) [reply nodefs/inode.go]: :)

**Comment by navytux** [comment]
Thanks.

**Comment by navytux** [comment]
> And tested before pushing. :)

Always a good idea.

**Pull request closed without merging**